tests/unit/integration/github/test_utils.py

```diff
@@ -73,7 +74,7 @@ def extract(self, text):
 
 def test_token_leak_disclosure_request_from_api_record():
     request = utils.TokenLeakDisclosureRequest.from_api_record(
-        {"type": "token", "token": "pypi-1234", "url": "http://example.com"}
+        {"type": "pypi_api_token", "token": "pypi-1234", "url": "http://example.com"}
```

Comment:
Hmmm, is this case sensitive? Docs indicate that these may be "screaming snake case" ala `PYPI_API_TOKEN`.

Reply:
disregard, I see the response in the issue <3